### DISCUSSION TITLE
How to list and filter only error type in telescope Workspace Diagnostics?

### DISCUSSION THREAD

**Discussion by erlangparasu** [LazyVim/LazyVim#3874] [Q&A]
How to list and filter only error type in telescope Workspace Diagnostics?

**Comment by folke** [comment]
https://github.com/folke/trouble.nvim?tab=readme-ov-file#-usage

And read the rest of the doc as well. You can also press `s` when diagnostics show to switch between severities

**Comment by erlangparasu** [comment]
i am calling Workspace Diagnostics using this default keymaps:
`{ "<leader>sD", "<cmd>FzfLua diagnostics_workspace<cr>", desc = "Workspace Diagnostics" },`

**Reply by erlangparasu** [reply]
How to call `Trouble diagnostics filter.severity = vim.diagnostic.severity.ERROR` while Telescope is open (text input is active) ?
